crowbar_framework/app/assets/stylesheets/fontawesome/_stacked.scss

```diff
@@ -0,0 +1,20 @@
+// Stacked Icons
+// -------------------------
+
+.#{$fa-css-prefix}-stack {
+  position: relative;
```

Comment:
Properties should be ordered display, height, line-height, position, vertical-align, width